src/Sentry.Unity/SentryOptionsConfiguration.cs

```diff
@@ -14,13 +14,26 @@ public class {{ScriptName}} : SentryOptionsConfiguration
             public override void Configure(SentryUnityOptions options)
             {
                 // Here you can programmatically modify the Sentry option properties used for the SDK's initialization
+
+        #if UNITY_ANDROID || UNITY_IOS
+                // Important!
+                // On Android and iOS, ALL options configured here will be "baked" into the exported project during
+                // build time. Any runtime changes to these options will not take effect.
+
+                // Examples:
+
+                // Works as expected and will disable all debug logging of the native SDKs
+                // options.Debug = false;
+
+                // Will NOT work as expected as this will need to get validated at runtime
+                // options.Debug = SystemInfo.deviceName.Contains("Pixel");
+        #endif
```

Comment:
The "will NOT WORK" note doesn't expand on what to do in that case. 

Not arguying this isnt' a step forward, just saying I fear it's still confusing

```suggestion
#if UNITY_ANDROID || UNITY_IOS
                // Important!
                // On Android and iOS, ALL options configured here will be "baked" into the exported project during
                // build time. Any runtime changes to these options will not take effect.

                // Examples:

                // Works as expected and will disable all debug logging of the native SDKs
                // options.Debug = false;

                // Will NOT work as expected as this will need to get validated at runtime
                // options.Debug = SystemInfo.deviceName.Contains("Pixel");
#endif
```